wlauto/workloads/androbench/__init__.py

```diff
@@ -0,0 +1,84 @@
+#    Copyright 2013-2015 ARM Limited
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+
+import os
+import re
+import time
+
+from wlauto import AndroidBenchmark
+from uiautomator import Device
+
+
+class Androbench(AndroidBenchmark):
+
+    name = 'androbench'
+    description = """Androbench measures the storage performance of device"""
+    package = 'com.andromeda.androbench2'
+    activity = '.main'
+    device=''
+
+    def setup(self, context):
+		global device
+		os.system('adb devices > deviceinfo')
+		devinf = open('deviceinfo','rb')
+		dev = devinf.readlines()[1].split('\t')[0]
+		devinf.close()
+		device=Device(dev)
+		os.system('rm deviceinfo')
```

Comment:
None of the above is necessary -- a connection to the device gets automatically intialized by the base class; UIAutomator interactions should be implemented in Java.